shopfloor/models/shopfloor_menu.py

```diff
@@ -2,51 +2,13 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 from odoo import _, api, exceptions, fields, models
 
-from odoo.addons.base_sparse_field.models.fields import Serialized
-
 
 class ShopfloorMenu(models.Model):
-    _name = "shopfloor.menu"
-    _description = "Menu displayed in the scanner application"
-    _order = "sequence"
-
-    _scenario_allowing_create_moves = (
-        "single_pack_transfer",
-        "location_content_transfer",
-    )
-
-    _scenario_allowing_unreserve_other_moves = (
-        "single_pack_transfer",
-        "location_content_transfer",
-    )
-
-    _scenario_allowing_ignore_no_putaway_available = (
-        "single_pack_transfer",
-        "location_content_transfer",
-    )
+    _inherit = "shopfloor.menu"
 
-    name = fields.Char(translate=True)
-    sequence = fields.Integer()
-    profile_id = fields.Many2one(
-        "shopfloor.profile", string="Profile", help="Visible on this profile only"
-    )
     picking_type_ids = fields.Many2many(
```

Comment:
I didn't find it in the view

Reply:
ahh, good catch thanks!

Reply:
fixed all missing fields / view override